packages/e2e-tests/src/specs/dashboard/templates/useTemplate.js

```diff
@@ -42,15 +42,42 @@ describe('Template', () => {
 
     await percySnapshot(page, 'Explore Templates');
 
-    await expect(firstTemplate).toClick('button', { text: 'Use template' });
+    await expect(firstTemplate).toClick('a', { text: 'See details' });
+    // Get count of template colors to compare to 'saved colors' in the editor.
```

Comment:
Would it be trivially possible to not only compare the count but also the actual colors?